app/tests/unit/forms/auth/middleware/apiAccess.spec.js

```diff
@@ -1,5 +1,5 @@
 const { getMockReq, getMockRes } = require('@jest-mock/express');
-const { v4: uuidv4 } = require('uuid');
+const uuid = require('uuid');
```

Comment:
For ease of understanding, will standardize on importing libraries and then explicitly using the library name and its method in the call. Will move away from this sort of thing where we make up a new name that is specific to this codebase.